docs/getting-started/intro-to-dockstore-tools-and-workflows.rst

```diff
@@ -1,27 +1,28 @@
-Introduction to Workflows, Tools, and Services
+Introduction to Workflows and Tools
 ==============================================
```

Comment:
I thought the linter enforced this, but I guess not (I'm confused actually, because I could swear I've seen itin the past), but we typically make the width of this line identical to the width of the line above it.

Reply:
The linter only enforces if the line is too short, not too long.